be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -1080,6 +1082,15 @@ Status HashJoinNode::_extract_join_column(Block& block, ColumnUInt8::MutablePtr&
                     raw_ptrs[i] = &col_nested;
                 }
             } else {
+                if constexpr (!BuildSide) {
+                    if (_join_op == TJoinOp::RIGHT_ANTI_JOIN &&
```

Comment:
Does only right anti join need to be converted?

Reply:
Yes,  right join and ful join will be converted in `_convert_block_to_null`.